services/spar/src/Spar/App.hs

```diff
@@ -169,7 +170,7 @@ createSamlUserWithId ::
   Sem r ()
 createSamlUserWithId teamid buid suid = do
   uname <- either (throwSparSem . SparBadUserName . cs) pure $ Intra.mkUserName Nothing (UrefOnly suid)
-  buid' <- BrigAccess.createSAML suid buid teamid uname ManagedByWire Nothing Nothing Nothing
+  buid' <- BrigAccess.createSAML suid buid teamid uname ManagedByWire Nothing Nothing Nothing defaultRole
```

Comment:
there is only one place where we call this function; better take a role as argument here and pass the default in there; that'll make it harder to forget about roles in future refactorings.